darts/models/forecasting/forecasting_model.py

```diff
@@ -353,6 +603,12 @@ def historical_forecasts(
             `series` that will be used as first prediction time.
             In case of ``pandas.Timestamp``, this time stamp will be used to determine the first prediction time
             directly.
+            If `start` is not specified, the first prediction time will automatically be set to :
+                 - the first predictable point if `retrain` is False
+
+                 - the first trainable point if `retrain` is True and `train_length` is None
+
+                 - the first trainable point + `train_length` otherwise
```

Comment:
Maybe a small sentence afterwards explaining precisely what is a "predictable point" and a "trainable point" might be welcome here (users may wonder).